### ISSUE TITLE
[Bug]: Pimcore X (10.1.0) - Deeplinks result in multiple redirects and end up with Error 500 

### ISSUE THREAD

**Issue by Fahl-Design** [pimcore/pimcore#9967]
### Expected behavior

Open the object in a new tab and not redirect to invalid Url

### Actual behavior

You go to https://demo.pimcore.fun/admin/login/deeplink?object_765_object
and end up here with error 500 

https://demo.pimcore.fun/admin/login/deeplink?1

![image](https://user-images.githubusercontent.com/6690962/128485872-b1836dc2-f86e-4900-a2a0-4deb5af2a4d6.png)

### Steps to reproduce

goto: https://demo.pimcore.fun/admin/login/deeplink?object_765_object

### Reason and Solution 
the generated "redirect" url is invalid because query parameter "deeplink" must be a sting like "true" not "1"

#### temp fix
create `templates/bundles/PimcoreAdminBundle/Admin/Login/deeplink.html.twig`

```twig
{% set redirect = path('pimcore_admin_login', {'deeplink': 'true'}) %}
<!DOCTYPE html>
<html>
    <head>
        <script src="/bundles/pimcoreadmin/js/pimcore/common.js"></script>
        <script src="/bundles/pimcoreadmin/js/pimcore/functions.js"></script>
        <script src="/bundles/pimcoreadmin/js/pimcore/helpers.js"></script>
        <script>
            {% if tab %}
                pimcore.helpers.clearOpenTab();
                pimcore.helpers.rememberOpenTab("{{ tab }}", true);
            {% endif %}
            window.location.href = "{{ redirect | raw }}";
        </script>
    </head>
    <body>
    </body>
</html>

```
the line
`{% set redirect = path('pimcore_admin_login', {'deeplink': 'true'}) %}`
fixes the invalid url
-> please fix "redirect" value in controller action

**Comment by dvesh3** [comment]
@Fahl-Design sorry, not able to reproduce with link provided. please clear cache and try again 😉 

Update: I cleared the cache and was able to reproduce the issue 😆

**Comment by brusch** [comment]
Fixed by #9972
